syntax = "proto3";

package chatroom.msgpb;

message AddFriend {
    uint32 user_id = 1;
    uint32 friend_id = 2;
}